pyproject.toml

```diff
@@ -37,6 +37,9 @@ classifiers = [
     "Topic :: Software Development :: Libraries :: Python Modules"
 ]
 
+[tool.poetry.urls]
+changelog = "https://python-poetry.org/history/"
```

Comment:
@neersighted I don't mind lowercase personally, but I think it's common to use capitalized words in the urls metadata, because the keys are displayed on PyPI website. I don't know if the website also does capitalization on its own.

E.g. https://pypi.org/project/mypy/

![image](https://user-images.githubusercontent.com/137616/195079793-0397497b-86ba-4e8a-8bd4-bcd397f57772.png)

Reply:
The example in packaging docmentation also uses capitalized keys: https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#project-urls

Reply:
Capital C works for me, then.